downgrade

```diff
@@ -4,6 +4,30 @@
 export TEXTDOMAIN=downgrade
 export TEXTDOMAINDIR=/usr/share/locale
 
+usage() {
+  cat <<EOF
+Usage: downgrade [options] [packages] [pacman_options]
+
+  Options:
+    --pacman <command>
+                    pacman command to use, defaults to "pacman"
+    --pacman-conf <directory>
+                    pacman configuration file, defaults to "/etc/pacman.conf"
+    --arch <architecture>
+                    target architecture, defaults to output of "uname -m"
+    --ala-url <url>
+                    location of ALA server, defaults to "https://archive.archlinux.org"
+    --ala           only use ALA server
+    --cached        only use cached packages
```

Comment:
I think it's ambiguous in the option from that `--x` means _only_ `x`. WDYT of either:

- Re-inverting these to `--no-x` to not use `x`, or
- Naming these `--only-x` or `--x-only`?

Reply:
Agree, changed them to variant `--x-only`